libs/libc/locale/lib_ctype.c

```diff
@@ -0,0 +1,81 @@
+/****************************************************************************
+ * libs/libc/locale/lib_ctype.c
```

Comment:
lib_ctype.c should be located in libs/libc/ctype.

Reply:
Got it!